game/server_core.cpp

```diff
@@ -282,7 +282,7 @@ void ServerCore::send_heartbeat()
     packet.state = this->state;
 
     size_t bufferSize = packet.calculateSize();
-    char *buffer = new char[CLIENT_RECV_BUFLEN];
+    char *buffer = new char[bufferSize];
```

Comment:
I think @hgrehm mentioned that we need to use CLIENT_RECV_BUFLEN

Reply:
I'm not familiar with the packet part, so do I replace the bufferSize for both ServerHeartbeatPacket and GameStatePacket with the same constant 'CLIENT_RECV_BUFLEN'?